Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM usgswma/python:3.8
 
 RUN apt-get install -y curl git
 
-RUN curl -sL https://deb.nodesource.com/setup_13.x | bash - && apt-get install -y nodejs
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -y nodejs
```

Comment:
Just a question, Is this to install node version 14? Would it be better to use 12 which is the LTS at the moment?

Reply:
Yes, it is installing 14. Hmm... an LTS would probably be less work to maintain.